HospitalToday/UI/MenuBuilder.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HospitalToday.UI
+{
+    class MenuBuilder
+    {
+        public MenuBuilder()
+        {
+            menuBuilder = new StringBuilder();
+            menuData = new MenuData();
+        }
+
+        private StringBuilder menuBuilder;
+        private MenuData menuData;
+
+        public int MainMenu()
+        {
+            Clear();
+            menuData.MainMenuData.ForEach(x => menuBuilder.AppendLine(x));
+            Console.WriteLine(menuBuilder);
+            int req;
+            Console.WriteLine("Answer:");
+            var answer = Console.ReadLine();
+            var result = int.TryParse(answer, out req);
```

Comment:
In general int is alias. It is better to use direct types for calling static members

Reply:
I changed alias on the full type name.